test.bash

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # prepare tests
-for f in check-*/test.sh check-*/test-*.sh
```

Comment:
There's no `test-*.sh` to be executed directly anymore

Reply:
[may][nit] or else: `check-*/test*.sh`

Reply:
I also wanted to allow each plugin to control how to run its tests, like avoiding port conflicts https://github.com/mackerelio/go-check-plugins/pull/629#discussion_r932153926.
It's still possible if `test.bash` executes all of `test*.sh`, but I think only executing `test.sh` is easier to control.